src/brs/db/sql/VersionedBatchEntitySqlTable.java

```diff
@@ -100,7 +108,21 @@ public void finish() {
             .execute();
         }
 
-        bulkInsert(txContext, getBatch().values());
+        /**
```

Comment:
maybe better to move the comment up ahead the update loop... this is a good explainer, why we are doing this